### ISSUE TITLE
Train efficientdet-d0 on COCO2017, and the model's performance are lower than yours.

### ISSUE THREAD

**Issue by Wanjpeng** [rwightman/efficientdet-pytorch#73]
First, Thanks to your great work!
I train efficientdet-d0 on COCO2017 with your latest code, and used your train script to train on 4xTITAN RTX.(train script:  ./distributed_train.sh 4 /mscoco --model efficientdet_d0 -b 22 --amp --lr .12 --sync-bn --opt fusedmomentum --warmup-epochs 5 --lr-noise 0.4 0.9 --model-ema --model-ema-decay 0.9999)
And I get about 32.2 mAP(about 122 epochs), then model seemed to become overfitting, here are my train logs:
[default_2020-08-28--21-29.log](https://github.com/rwightman/efficientdet-pytorch/files/5146796/default_2020-08-28--21-29.log)
and summary.csv(I add ".log" to summary file for upload to this issue.):
[summary.csv.log](https://github.com/rwightman/efficientdet-pytorch/files/5146800/summary.csv.log)
Could you please help me solving this problem or tell me what I did wrong?
Thanks very much sincerely.

**Comment by rwightman** [comment]
you stopped it too soon

**Comment by Wanjpeng** [comment]
Thanks for your reply, I will continue train and see what it will go on.

**Comment by Wanjpeng** [comment]
@rwightman Thanks, now I get about 33.6mAP around 210 epochs.
